### DISCUSSION TITLE
Opening direct link don't work on server

### DISCUSSION THREAD

**Discussion by silicology1** [leptos-rs/leptos#2559] [Q&A]
When I open dist folder using server
```sh
python3 -m http.server
```

The direct router link don't open, it gives error 
http://localhost:8000/add-profile
```
Error response

Error code: 404

Message: File not found.

Error code explanation: HTTPStatus.NOT_FOUND - Nothing matches the given URI.

```

But it works fine with trunk serve.

How can I fix it?

**Comment by gbj** [comment]
You need to rewrite any unmatched routes to `/index.html` This tends to be the case for any single-page app with client-side routing. Not sure how to do it with the python server but I'm sure it's possible to find online. Look for how you'd do the same thing for React, Vue, Angular, etc.

**Comment by silicology1** [comment]
I successfully [installed](https://create-react-app.dev/docs/deployment/) on fleek.  

_redirects
```
/*  /index.html  200
```
just need _redirects file in it. Thanks.
